runtime/gc_base/modronapi.cpp

```diff
@@ -51,6 +51,20 @@
 
 extern "C" {
 
+void
+preMountContinuation(J9VMThread *vmThread, j9object_t object)
+{
+	/*  */
+
+}
+
+void
+postDismountContinuation(J9VMThread *vmThread, j9object_t object)
+{
+	/* blindly remember the continuation Object via J9WriteBarrierBatch. */
```

Comment:
This is not just about remembering. A more generic comment would more like: 
Conservatively assume that via mutations of stack slots (which are not subject to access barriers), all post-write barriers have been triggered on this Continuation object, since it's been mounted.